cli/lsp/language_server.rs

```diff
@@ -2060,13 +2061,7 @@ impl Inner {
           action_data.action_name,
         )
         .await?;
-      code_action.edit = refactor_edit_info
-        .to_workspace_edit(self)
-        .await
-        .map_err(|err| {
-          error!("Unable to convert changes to edits: {}", err);
-          LspError::internal_error()
-        })?;
```

Comment:
Any particular reason to remove this additional context?

Reply:
It's still there, just moved inside of `to_workspace_edit()`.